Sources/OAuthenticator/Authenticator.swift

```diff
@@ -157,6 +158,14 @@ public final class Authenticator {
 	}
 }
 
+/// A default `URLSession`-backed `URLResponseProvider`.
+@MainActor
+public let defaultAuthenticatorResponseProvider: URLResponseProvider = {
+	let session = URLSession(configuration: .default)
+
+	return session.responseProvider
+}()
```

Comment:
Was this moved from `Authentication.defaultResponseProvider`?

Reply:
Unfortunately, yes. I would be happy to hear of a way to avoid this. Currently, Swift does not support statically stored properties in generic types. I know this would be source breaking.